src/style.css

```diff
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html * {
+  border: none;
+  outline: none;
+  background-color: #fff0;
+  font-family: Avenir, sans-serif;
+}
+
+.searchContainer {
+  box-sizing: border-box;
+  margin-top: 20px;
+  min-width: 200px;
+  background-color: #ffff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchContainer--big {
+  height: 70px;
+}
+
+.searchContainer--small {
+  height: 42px;
+}
+
+.searchIcon--big {
+  width: 19px;
+  height: 19px;
+  margin: 24px 18px 0 25px;
+  float: left;
+}
+
+.searchIcon--small {
+  width: 11px;
+  height: 11px;
+  margin: 14px 10px 0 12px;
+  float: left;
+}
+
+.searchContainer--big:hover,
+.searchContainer--small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchContainer--big:focus-within {
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.searchContainer--small:focus-within {
+  box-shadow: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
```

Comment:
here is difference only in `box-shadow` and `text-shadow` in this classes. Create some common class where you can add similar styles

Reply:
How  is that possible to create class for pseudo elements :hover and :focus ?

Reply:
I think he meant you should combine it into common selector

Reply:
Thank you, now I get it. Similar styles added to .searchContainer with state hover and focus.